apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx

```diff
@@ -90,9 +94,24 @@ const styles = StyleSheet.create({
     padding: 12,
   },
   text: {
-    fontFamily: 'Rubik-Medium',
     fontSize: 14,
-    fontWeight: '500',
     lineHeight: 22,
   },
 })
+
+const bold = {b: (text) => <Text bold>{text}</Text>}
+
+export const useLocalStrings = () => {
+  const intl = useIntl()
+
+  return {
+    manyNameServersWarning: intl.formatMessage(messages.manyNameServersWarning, bold),
```

Comment:
add to the `useStrings` of the funnel, and return as a function that takes the a react node.

Reply:
done